FontAwesome/FontAwesome.swift

```diff
@@ -33,18 +33,7 @@ public extension UIFont {
     /// - parameter fontSize: The preferred font size.
     /// - returns: A UIFont object of FontAwesome.
     public class func fontAwesomeOfSize(fontSize: CGFloat) -> UIFont {
-        struct Static {
-            static var onceToken : dispatch_once_t = 0
-        }
-
-        let name = "FontAwesome"
-        if UIFont.fontNamesForFamilyName(name).isEmpty {
```

Comment:
Would it be so that when you DO add the font in the main bundle, this `if` statement would fail, since it is already there? And the return would return the font from the main bundle?
(just asking, did not test this....)